src/roadmap/data/upcoming.py

```diff
@@ -113,7 +113,7 @@
     },
     {
         "name": "MariaDB 11.0",
-        "type": "addition",
+        "type": "Addition",
```

Comment:
This seems like presentation that should be handled in the UI rather than data coming from the backend.

Reply:
I can change the frontend to also accept lowercase data but for consistency sake all the other `type` fields in the mocks are capitalized.

Reply:
Oh, I see what you mean. They should probably all be lowercase from the API and capitalized for consistency by the UI. The caller should assume it will need to formate the response data in some way. I don't think case is part of the schema.